popup.js

```diff
@@ -51,14 +53,17 @@ function assingDateToElement(dateOnly, element, fullDate) {
 }
 
 function assignDataFromMsg(lastMsg) {
+    if (!('data' in lastMsg))
```

Comment:
Proste to tez by było git ale to nie jest konieczne :) Jak dla mnie wszystko piękne można mergowac. 
```suggestion
    if (!lastMsg.data) {
```